autogluon/training/docker/0.3/py3/Dockerfile.cpu

```diff
@@ -55,7 +56,17 @@ RUN apt-get update \
  && apt-get autoremove -y \
  && apt-get clean \
  && rm -rf /var/lib/apt/lists/*
- 
+
+# install OpenSSL
```

Comment:
Is there a reason we can't install openSSL from apt-get?

Reply:
apt-get doesn't update openssl probably because it's manually installed in base images.